rdb/Makefile.am

```diff
@@ -0,0 +1,25 @@
+AM_CXXFLAGS = -Wno-error=format
+AM_CFLAGS = -Wno-error=format
+bin_PROGRAMS = rdb-cli$(EXEEXT)
+rdb_cli_SOURCES =
+
+clone-repo:
```

Comment:
Why this Rd tool needs to be installed here ? It seems like a bad design, it should be handled in Sonic build image repo in docket build script

Reply:
According to Qiluo's review comments below. The Rdb-cli should only be installed into the syncd container. Sonic-sairedis's files, like saidump.sh, saidump are installed into the syncd container too. So it's a choice to install rdb-cli here.
https://github.com/sonic-net/sonic-buildimage/pull/19268#discussion_r1697663604

In the folder sonic-buildimage/dockers/ shown below, I could not find a script that can be used only to install rdb-cli into the syncd container. Could you give some suggestions? Thanks.

https://github.com/sonic-net/sonic-buildimage/tree/master/dockers

Reply:
@kcudnik, @qiluo-msft , how about modifying the below file?
https://github.com/sonic-net/sonic-buildimage/blob/master/platform/broadcom/docker-syncd-brcm-dnx/Dockerfile.j2

Like adding below commands:
```
# Obtain librdb source codes
RUN git clone https://github.com/redis/librdb.git

# Build librdb
RUN cd ./librdb/;git submodule update --init --recursive;make -j4;cd ..

# Install rdb-cli
RUN install -p -D -m 0775 ./librdb/bin/rdb-cli /usr/bin/rdb-cli

# Clear librdb source codes
RUN rm -rf ./librdb
```